setup.py

```diff
@@ -97,7 +97,7 @@
     # bug in pip: matplotib is ignored here because it is already present in
     # install_requires.
     "tests": ["pytest>=3.6", "pytest-mpl", "pytest-xdist", "pytest-rerunfailures", "pytest-instafail", "matplotlib>=3.1"],
-    "coverage":["pytest-cov", "codecov"],
```

Comment:
I'm not too familiar with `codecov`, guessing `pytest-cov` supersedes it?

Reply:
https://github.com/codecov/codecov-python: as it says in the readme, it is deprecated.